src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -205,12 +205,6 @@ export default class DefineRegistrationSummary extends Mixins(DateMixin) {
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
-.define-registration-step-error-message {
-  padding-top: 1.25rem;
-  padding-left: 1.25rem;
-  color: $app-red;
-}
```

Comment:
The error message is still red, as before. I guess this color style wasn't needed.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/17c907d4-be8c-43f5-8028-cfe54ed8cb4c)